components/Navbar.js

```diff
@@ -0,0 +1,87 @@
+import Link from 'next/link'
```

Comment:
I think this file is included by mistake

Reply:
Correctomundo, will fix